compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/JvmSymbols.kt

```diff
@@ -723,6 +727,16 @@ class JvmSymbols(
 
     val runSuspendFunction: IrSimpleFunctionSymbol =
         kotlinCoroutinesJvmInternalRunSuspendKt.functionByName("runSuspend")
+
+    val rawTypeAnnotationClass: IrClassSymbol =
```

Comment:
It would be great to use the same IR class symbol instance as in psi2ir, at least to avoid code duplication. I propose to pass the `JvmGeneratorExtensions` instance to `JvmBackendContext` (instead of `classNameOverride`), and use `context.generatorExtensions.rawTypeAnnotationConstructor` in `rawType`.

Reply:
Good point, I didn't realize that we're already passing a part of the generator extensions to the backend context. Done.